client/src/components/ElementEditor/Header.js

```diff
@@ -209,7 +195,7 @@ class Header extends Component {
             </Tooltip>}
           </div>
           <h3 className={titleClasses}>{title}</h3>
-          {this.renderStatusBadge()}
+          {this.renderStatusFlagBadges()}
```

Comment:
We're rendering potentially multiple badges now, so I've updated the logic to handle that and to pull the status flags from `BaseElement`.